test/remote/db_tests.js

```diff
@@ -183,6 +184,19 @@ describe('remote db', function() {
           assert.equal(sessions[0].uaFormFactor, null, 'uaFormFactor property is correct')
           assert.equal(sessions[0].lastAccessTime, sessions[0].createdAt, 'lastAccessTime property is correct')
 
+          // Simulate a redis error
```

Comment:
Not for this PR, but some tech debt day. There are a couple test `it` cases in here that are 300+ lines of code and could be refactored a bit.